website/tutorial/index.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 1
+title: Introduction
+description: Introduction to the tutorials section
+keywords: [podman desktop, podman, tutorial]
+tags: [podman-desktop, tutorial]
+---
+
+# Introduction
+
+This tutorial covers end-to-end tasks that you must perform to accomplish a goal while working with the Podman Desktop application. Each tutorial section includes one or more procedures that contain a sequence of steps to follow.
+
+Before going through each tutorial, get yourself familiarize with the key UI components of the Podman Desktop application:
+
```

Comment:
I think I'm lost if here it's saying tutorial covers end-to-end tasks then tutorial section and then list of tutorials.

`This tutorial covers` --> `These tutorials overs`

Then instead of 'Each tutorial section' --> 'Each tutorial' as for me a tutorial is a section